tasks/create_vpc_route_table.yml

```diff
@@ -18,13 +20,20 @@
       # type must be present
       - item.type is defined
       # type must be one of the following
-      - item.type in ['igw', 'nat', 'vgw', 'instance', 'interface', 'peering']
+      - item.type in ['igw', 'nat', 'vgw', 'instance', 'eni', 'peering']
       # igw does not require a 'filter' or 'name', the rest do
-      - (item.type in ['igw']) or (item.type in ['nat', 'vgw', 'instance', 'interface', 'peering'] and (item.filter is defined or item.name is defined))
+      - >-
+        (item.type in ['igw']) or
```

Comment:
why not using the equal operand here, for one match only?

Reply:
This is actually true ;p